src/main/resources/application.properties

```diff
@@ -10,7 +10,15 @@ logging.level.gov.nasa.pds.api.engineering.elasticsearch = DEBUG
 logging.level.gov.nasa.pds.api.engineering.controllers = DEBUG
 
 ## note the port is mandatory even when it is default :80 or :443
-elasticSearch.host=localhost:9200
+#elasticSearch.host=localhost:9200
+#elasticSearch.registryIndex=registry
+#elasticSearch.registryRefIndex=registry-refs
+#elasticSearch.timeOutSeconds=60
+#elasticSearch.username=
+#elasticSearch.password=
+#elasticSearch.ssl=false
+
+elasticSearch.host=search-pds-dev-esext-kcq7xxa4lsrakjw33lywpjdyfy.us-west-2.es.amazonaws.com:443
```

Comment:
Can we set `localhost:9200` as default, so developers can run the app from Eclipse or other IDE without changing this file.